group_vars/matrix_servers

```diff
@@ -729,6 +729,27 @@ matrix_bot_matrix_reminder_bot_database_password: "{{ matrix_synapse_macaroon_se
 ######################################################################
 
 
+######################################################################
+#
+# matrix-bot-go-neb
+#
+######################################################################
+
+# We don't enable bots by default.
+matrix_bot_go_neb_enabled: false
+
+matrix_bot_go_neb_systemd_required_services_list: |
+  {{
+    ['docker.service']
+  }}
+
```

Comment:
Here would be a good place to redefine the value of `matrix_bot_go_neb_container_http_host_bind_port` like this:

```yaml
matrix_bot_go_neb_container_http_host_bind_port: "{{ '' if matrix_nginx_proxy_enabled else '127.0.0.1:4050' }}"
```